<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFPlayClass</class>
 <widget class="QWidget" name="QFPlayClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>稀饭大大的播放器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="QFPlay.qrc">
    <normaloff>:/QFPlay/Resources/logo.ico</normaloff>:/QFPlay/Resources/logo.ico</iconset>
  </property>
  <widget class="VideoWidget" name="openGLWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="openButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>500</y>
     <width>56</width>
     <height>56</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:!hover{border-image: url(:/QFPlay/Resources/open_normal.png);}
QPushButton:hover{border-image: url(:/QFPlay/Resources/open_hot.png);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="playButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>500</y>
     <width>56</width>
     <height>56</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:!hover{border-image: url(:/QFPlay/Resources/play_normal.png);}
QPushButton:hover{border-image: url(:/QFPlay/Resources/play_hot.png);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="totaltime">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>530</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255); 
font:16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="playtime">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>530</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255); 
font:16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>530</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255); 
font:16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QFSlider" name="playslider">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>470</y>
     <width>641</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>videowidget.h</header>
  </customwidget>
  <customwidget>
   <class>QFSlider</class>
   <extends>QSlider</extends>
   <header>qfslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="QFPlay.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>openButton</sender>
   <signal>clicked()</signal>
   <receiver>QFPlayClass</receiver>
   <slot>open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>538</y>
    </hint>
    <hint type="destinationlabel">
     <x>837</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playslider</sender>
   <signal>sliderPressed()</signal>
   <receiver>QFPlayClass</receiver>
   <slot>sliderPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>581</x>
     <y>475</y>
    </hint>
    <hint type="destinationlabel">
     <x>955</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playslider</sender>
   <signal>sliderReleased()</signal>
   <receiver>QFPlayClass</receiver>
   <slot>sliderRelease()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>641</x>
     <y>471</y>
    </hint>
    <hint type="destinationlabel">
     <x>1004</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playButton</sender>
   <signal>clicked()</signal>
   <receiver>QFPlayClass</receiver>
   <slot>play()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>505</x>
     <y>524</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>-14</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>open()</slot>
  <slot>sliderPress()</slot>
  <slot>sliderRelease()</slot>
  <slot>play()</slot>
 </slots>
</ui>
